server/initializers/constants.ts

```diff
@@ -155,10 +157,12 @@ const JOB_ATTEMPTS: { [id in JobType]: number } = {
   'video-live-ending': 1,
   'video-studio-edition': 1,
   'manage-video-torrent': 1,
-  'move-to-object-storage': 3
+  'move-to-object-storage': 3,
+  'video-channels-sync': 1,
+  'video-channel-import': 3
```

Comment:
About the naming:
 - I am not sure what `video-channel-do-sync` and `video-channels-check-sync` would refer? ( `video-channel-do-sync` would refer to `video-channels-sync` and `video-channels-check-sync` would refer to `video-channel-import`?)
 - I still think the naming of `video-channel-import` may be good as it could be reused for other uses (importing a whole video channel without syncing it).
 - Maybe `video-channel-sync-latest` (`channel` in singular and `latest` to refer to latest publications)

But as you are the maintainer of the application, I will follow your suggestion if you're not convinced of my explanation anyway :).

About the scheduler, I assume you refer to the job currently named `video-channels-sync`. Indeed, it will probably ease my job and the testing

Reply:
> I am not sure what video-channel-do-sync and video-channels-check-sync would refer? ( video-channel-do-sync would refer to video-channels-sync and video-channels-check-sync would refer to video-channel-import?)

No, it's just another name for `video-channels-sync`. My initial goal is to avoid confusion with the `video-channel-import` task
`video-channel-sync-latest` seems perfect